Xamarin.Forms.Core/TouchState.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	[Flags]
+	public enum TouchState
+	{
+		Default = 0 << 0,
+		Pressed = 1 << 1,
+		Released = 1 << 2,
+		Move = 1 << 3,
+		Cancelled = 1 << 4,
+		Failed = 1 << 5,
+		Changed = 1 << 6,
+		Entered = 1 << 7,
+		Exited = 1 << 8,
+		Hover = 1 << 9
+	}
+
+	public static class TouchStateExtensions
```

Comment:
Should this public on intenal? maybe move it to it's own class, it could grow.

Reply:
done